source/help/getting-started/organizing-conversations.rst

```diff
@@ -154,7 +154,9 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
-**Drag and drop channels and categories**: Drag channels between or within categories, or drag to reorder entire categories to prioritize important conversations.
+**Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). To clear channel selections, press ESC.
```

Comment:
0/5 I think this should fall under the feature of **Drag and drop channels and categories** rather than being placed here

Reply:
@esethna Once multiple channels are selected, end users can drag and drop those selected channels elsewhere, or they can use the **More options** menu to move those selected channels. As a result, I positioned the selection options ahead of the move options in the product documentation (to match order of operations).

Thoughts?